www/source/partials/docs/_internals-bootstrapping.html.md.erb

```diff
@@ -5,164 +5,339 @@ This document provides developer documentation on how the Habitat system becomes
-$ rm -f /src/plans/tmp/${DB_PREFIX}*.db
-$ cd /src/plans
+# (Optional) run all test suites in do_check() build phases
+$ export DO_CHECK=true
+# Build all base plans in order
+$ ./core-plans/bin/bootstrap/stage1-build-base-plans.sh
+```
+
+Once the set has finished building, we are going to use the newly built Studio immediately and move the artifacts to the side for more use:
+
+```shell
+# Exit the Studio
+$ exit
+# Save the stage1 artifacts
+$ mv ./results ./results-stage1
+# Install the just-built Studio
+$ sudo hab install ./results-stage1/core-hab-studio-*.hart
+```
+
+Now we're ready to use the stage1 packages to make a set of packages that are completely decoupled from the stage1 `tools/` infrastructure, which we'll call a "stage2" build.
+
+## Part V: Stage 2
+
+In this stage, we rebuild all the base packages needed by Habitat using the tools (compiler, etc.) from the previous stage, thus making the system self-hosted. This phase involves not using existing packages that are published in any existing Builder API, so we're going to more carefully create a `default`-type Studio. If we use a fully-qualified package identifier and use the `HAB_STUDIO_BACKLINE_PKG` environment variable, this will skip the call back to a Builder instance.
+
+To make this work, we need to get the fully-qualified package identifier of the `core/hab-backline` package we just built in stage1. You can use `ls -1 ./results-stage1/core-hab-backline-*.hart` to list the artifact. The naming of this file will help you to figure out the package identifier. For example this file:
+
+```
+./results-stage1/core-hab-backline-0.56.0-20180322160653-x86_64-linux.hart
+```
+
+Has a package identifier of `core/hab-backline/0.56.0/20180322160653`. In general, the `/` characters in a package identifier become `-` characters in an artifact name, and you drop the target info (i.e. `x86_64`) and file extension. For the remainder of this guide, let's assume that your fully-qualified package identifier is `core/hab-backline/0.56.0/20180322160653`.
+
+Finally, to help set up environment variables for the Studio, we're going to use a [stage2 Studio](https://github.com/habitat-sh/core-plans/blob/master/bin/bootstrap/stage2-studio.sh) wrapper:
 
-$ time record ${DB_PREFIX}base make base
+```shell
+# Install only stage1 packages when setting up this Studio--that's why we're
+# calling `new` explicitly before using
+$ env HAB_STUDIO_BACKLINE_PKG=core/hab-backline/0.56.0/20180322160653 ./core-plans/bin/bootstrap/stage2-studio.sh new
+# Enter the stage2 Studio
+$ ./core-plans/bin/bootstrap/stage2-studio.sh enter
 ```
 
+Now that we're in a Studio, it's been set up with an empty artifact cache. We're going to set things up so that our install logic won't need to call out to a Builder API:
+
 ```studio
-$ $BUILD libarchive
-$ $BUILD libsodium
-$ $BUILD zeromq
-$ $BUILD ../components/depot
+# Copy all stage1 packages into the artifact cache
+$ cp ./results-stage1/*.hart /hab/cache/artifacts/
```

Comment:
The `*` character does not appear on acceptance. Maybe the template engine swallows it? It's not practical for copy/pasting lines :) this happens in several places in code snippets.

Reply:
Oooh, that's not good 😦 

My guess is that it's due to the templating engine, similar to https://github.com/habitat-sh/habitat/issues/4587.

Wonder if `\*` works here?

Pinging @cnunciato for other ideas.